spec/features/taxonomy_generation/generate_terms_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "rails_helper"
+
+RSpec.feature "Generating terms", type: :feature do
+  it "allows users to generate terms" do
+    given_theres_a_project_with_todos
+
+    when_i_visit_the_taxonomy_project_page
+    and_i_click_to_start_with_a_project
+    then_i_should_see_a_page
+    when_i_submit_a_term
+    then_the_term_is_saved
+    and_i_see_the_next_page
+
+    when_i_go_to_the_project_page
+    then_i_see_the_generated_terms
+  end
```

Comment:
Cool! We started experimenting with it in content-tagger, and it has held up pretty well. 

We got it from FutureLearn: https://about.futurelearn.com/blog/how-we-write-readable-feature-tests-with-rspec/